client/.eslintrc.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
     'no-param-reassign': 0,
     'no-mixed-operators': 0,
     'no-underscore-dangle': 0,
+    "object-curly-newline": "off",
```

Comment:
What a nuisance this is.. doesn't allow:
```js
const { var1, var2, var4, var4, var5 } = this.props;
```

Reply:
It requires newlines if there are too much fields (IIRC, 4 and more) :slightly_smiling_face:

Reply:
Yeah, it makes no sense.. that's what we got `max-len` for.